extension/src/experiments/model/collect.ts

```diff
@@ -312,19 +317,51 @@ const collectFromBranchesObject = (
   }
 }
 
+const getCommitDataFromOutput = (
+  output: string
+): CommitData & { hash: string } => {
+  const data: CommitData & { hash: string } = {
+    author: '',
+    date: '',
+    hash: '',
+    message: '',
+    tags: []
+  }
+  const [hash, author, date, refNamesWithKey] = output
+    .split('\n')
+    .filter(Boolean)
+  data.hash = hash
+  data.author = author
+  data.date = date
+
+  const message = output.match(/\nmessage:(.+)/s) || []
```

Comment:
Collecting the message and the tag names ended up being a little more complicated than the other items (see previous comment). Any ideas on simplifying things are welcome!